test/filter_design.jl

```diff
@@ -1012,6 +1012,13 @@ end
     winfirtaps_scipy = readdlm(joinpath(dirname(@__FILE__), "data", "digitalfilter_hamming_129_bandpass_fc0.1_0.2_fs1.0.txt"),'\t')
     @test winfirtaps_jl ≈ winfirtaps_scipy
 
+    @test_throws ArgumentError ComplexBandpass(2, 1)
+
+    # TODO: write actual tests for ComplexBandpass
+    winfirtaps_jl = digitalfilter(ComplexBandpass(0.1, 0.2), FIRWindow(hamming(128), scale=false); fs=1)
+    winfirtaps_scipy = readdlm(joinpath(dirname(@__FILE__), "data", "digitalfilter_hamming_128_complexbandpass_fc1.txt"), '\t', ComplexF64)
+    @test winfirtaps_jl ≈ winfirtaps_scipy  # replace with real test data
```

Comment:
Can we add a comment how the reference data was generated here? Also, I think we should check with another `fs`, just to make sure the scaling works as expected, e.g.
```julia
digitalfilter(ComplexBandpass(100, 200), FIRWindow(hamming(128), scale=false); fs=1000)
```
should give the same result.

Reply:
Ah, I wasn't aware of this circulus vitiosus. (Even noting the comment, the `_scipy` suffix in the variable name lead me to wrong conclusions.)

I'll try to cook up a more effective test.